packages/web/src/components/range/RangeInput.js

```diff
@@ -214,8 +205,6 @@ RangeInput.propTypes = {
 	themePreset: types.themePreset,
 	componentId: types.stringRequired,
 	includeNullValues: types.bool,
-	enableAppbase: types.bool,
```

Comment:
@mohdashraf010897 We should changes carefully

Reply:
done 👍